snapcraft/storeapi/__init__.py

```diff
@@ -503,6 +521,28 @@ def push_snap_build(self, snap_id, snap_build):
         if not response.ok:
             raise errors.StoreSnapBuildError(response)
 
+    def snap_status(self, snap_id, series, arch):
+        qs = {}
+        if series:
+            qs['series'] = series
+        if arch:
+            qs['arch'] = arch
+        url = 'snaps/' + snap_id + '/status'
```

Comment:
Nevermind. If we ever run this in windows, the path will be wrong. Hardcoding the / is the right thing to do.